src/main/java/de/hysky/skyblocker/utils/ItemUtils.java

```diff
@@ -198,6 +199,10 @@ public static IntIntPair getDurability(@NotNull ItemStack stack) {
         return null;
     }
 
+    /**
+     * Gets the first line of the lore that matches the specified predicate.
+     * @return The first line of the lore that matches the predicate, or {@code null} if no line matches.
+     */
     @Nullable
     public static String getLoreLineIf(ItemStack item, Predicate<String> predicate) {
```

Comment:
Idk who wrote this originally but `ItemStack` instances should be `stack`, not `item`.

Reply:
Done in [`3f1b9d9`](https://github.com/SkyblockerMod/Skyblocker/pull/728/commits/3f1b9d9842018c905b29a763e018916ee3dcabe3).